### ISSUE TITLE
CSS classes are too generic

### ISSUE THREAD

**Issue by cmiller86** [ocombe/ocModal#10]
ocModal is using generic css classes like .modal which overrides the default rules for all modals. Prepending oc- on the classes would avoid class conflicts.

**Comment by ocombe** [comment]
True, but the idea was to make it work with bootstrap when I made it (and bootstrap uses .modal).
It could be a config param to make it add the modal class when using bootstrap (or not).

**Comment by cmiller86** [comment]
It currently causes an issue if you have any other bootstrap modals on the page. I have existing modals that are display none but the css for ocModal changes their display to block so they're no longer hidden. Ideally, the ocModal's modal html would use something like class="modal oc-modal" with the custom css targeted at the oc-modal class instead of the default bootstrap class. Bootstrap's .modal class default is display: hidden.

**Comment by benjamincharity** [comment]
> Ideally, the ocModal's modal html would use something like class="modal oc-modal" with the custom css targeted at the oc-modal class instead of the default bootstrap class.

Definitely agree with @cmiller86 on this. Unless this library depends on Bootstrap, it seems there should be a separation of concerns.

(Freaking love this library btw!)
